index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+    <style>
+    @media (max-width:700px)  { /* portrait e-readers (Nook/Kindle), smaller tablets @ 600 or @ 640 wide. */ 
+    #sec2{
+        text-align: left;
+    }
+    #header{
+      display: flex;
+    }
+
+}
+    </style>
```

Comment:
- These lines are repeated in your CSS file. Kindly use your CSS file.
- Would be great to have CSS styles in a CSS file.